lblod/app-worship-decisions-database
# Adding new columns to show EB and CB org:hasSubOrganization relationship

# Description
DL-5013

This PR adds `?worshipAdministrativeUnitRelationship` and `?worshipAdministrativeUnitRelationshipLabel` in report `link-between-worship-service-and-admin-units-report`. These variables will link worship administrative units with the rule EB has CB and show them in the report with the OPTIONAL statement. 

Report description is also changed in https://github.com/lblod/app-worship-decisions-database/pull/72/commits/3dba970c8fbc61bc3c87f937ce0dcf9d297a1b52

# Type of change


# Related services

- report-generation

# How to test 

1. Add in `docker-compose.override.yml` 
```yml
  report-generation:
    links:
      - virtuoso:database  # we need to because performance
    ports:
      - 9999:80
```
3. `drc up -d`
4. Links between worship services and non-religious administrative units → 
```bash 
curl -X POST -H "Content-Type: application/vnd.api+json"  -d '{  "data": { "attributes": { "reportName": "linksBetweenWorshipServicesAndAdminUnitsReport" }  }  }'  http://localhost:9999/reports
```
5. Generated report = `{"data":{"type":"report-generation-tasks","attributes":{"status":"success"}}}` and should be visible in `/data/files/`

# What to check

- N/A

# Links to other PR's

- N/A

# Notes

- Update changelog
